source/features/clean-conversation-headers.tsx

```diff
@@ -73,7 +49,6 @@ void features.add(import.meta.url, {
 	additionalListeners: [
 		onConversationHeaderUpdate,
 	],
-	awaitDomReady: false,
```

Comment:
This makes the feature very unreliable, so I dropped it (and the calls to `elementReady` as well).

Reply:
But this makes the feature slower, we don't really want to show these elements until the whole conversation has loaded